packages/frontend/src/hooks/useSavedQuery.ts

```diff
@@ -160,9 +160,11 @@ export const useUpdateMultipleMutation = (projectUuid: string) => {
 };
 
 export const useUpdateMutation = (savedQueryUuid?: string) => {
```

Comment:
this is being used in one more place. I would extract both as I've seen using code like this produce bugs previously

```
    const { projectUuid } = useParams<{ projectUuid: string }>();
    const dashboardUuid = useSearchParams('fromDashboard');
```

out of this hook and input as params